.travis.yml

```diff
@@ -6,9 +6,6 @@ rvm:
   - 2.6
   - jruby-head
 
-jdk:
-  - oraclejdk8
-
```

Comment:
Why the removal?

Reply:
The Travis builds on my fork failed as it couldn't install the package:
https://travis-ci.org/ashmaroli/kramdown/jobs/532961008

Reply:
I have update the whole `.travis.yml` file and it seems to work fine now...